api/openapi.yaml

```diff
@@ -2346,6 +2342,14 @@ components:
         - customize
       example: customize
       type: string
+    KernelFileTypes:
+      description: File name of the kernel file, `Image` for aarch64. 'vmlinuz' for x86. Defaults to the respective architecture type.
+      example: vmlinuz
+      enum:
```

Comment:
I don't know that we can limit it to just these two. There are at three that we deal with in other places in IMS:
https://github.com/Cray-HPE/image-recipes/blob/develop/create_init_ims_manifest.py#L33-L40
I don't know if anyone uses custom kernel file names...

Reply:
Okay I think in this case we can just allow the user to specify any name, and we can just check the default mappings such as x86 - vmlinuz, aarch64 - Image. If no matches are found with a custom kernel file name then we just keep it as they passed in?